debian/content.md

```diff
@@ -8,6 +8,8 @@ Debian is an operating system which is composed primarily of free and open-sourc
 
 # About this image
 
+The images in this repository are intended to be as minimal as possible (because of the immutable/layered nature of containers, it's much easier to add than it is to remove). More specifically, they're built from [the "minbase" variant](https://manpages.debian.org/stable/debootstrap/debootstrap.8.en.html#variant=minbase_buildd_fakechroot), which only installs "required" packages, and thus creates the smallest possible footprint that is still "Debian" (as defined/managed by [the Release and FTP teams](https://www.debian.org/intro/organization#distribution) within the project).
```

Comment:
This may add a nice highlight effect to really call it out.

```suggestion
> [!NOTE]
> The images in this repository are intended to be as minimal as possible (because of the
immutable/layered nature of containers, it's much easier to add than it is to remove). More specifically, they're built from [the "minbase" variant](https://manpages.debian.org/stable/debootstrap/debootstrap.8.en.html#variant=minbase_buildd_fakechroot), which only installs "required" packages, and thus creates the smallest possible footprint that is still "Debian" (as defined/managed by [the Release and FTP teams](https://www.debian.org/intro/organization#distribution) within the project).
```

Reply:
Unfortunately the target of this is Docker Hub, not GitHub, and that's a GitHub-specific markup.